### ISSUE TITLE
`set_active_*` have inconsistent default `preference`

### ISSUE THREAD

**Issue by MatthewFlamm** [pyvista/pyvista#2515]
**Describe what maintenance you would like added.**

As noted in #2500 in [comment](https://github.com/pyvista/pyvista/pull/2500#discussion_r859040768)

https://github.com/pyvista/pyvista/blob/ce1a3741b541d91fd59fa7cda0984fc43732fa0f/pyvista/core/dataset.py#L622

https://github.com/pyvista/pyvista/blob/ce1a3741b541d91fd59fa7cda0984fc43732fa0f/pyvista/core/dataset.py#L662

https://github.com/pyvista/pyvista/blob/ce1a3741b541d91fd59fa7cda0984fc43732fa0f/pyvista/core/dataset.py#L697

-----

**Comment by adeak** [comment]
To carry over my comment from the PR:

There are 15 instances of `preference='point'` (or with double quotes) and 10 instances of `preference='cell'` (or with double quotes) in `pyvista/`. This excludes docstring cameos, but includes code inside method bodies (so these numbers don't reflect the respective default parameters)...

**Comment by adeak** [comment]
> There are probably use cases where it make sense to change the preference, e.g. if a filter only operates on cell data for example, but this should be the exception and documented as such. This assumes that point data is chosen as the overall default, but this does not imply endorsement of that position.

Good point, thanks. I haven't checked context beyond excluding doctest lines (`>>>` or `...` prefix somewhere along the line) from the output of
```sh
git grep 'preference="point"\|'"preference='point'" pyvista/
# vs
git grep 'preference="cell"\|'"preference='cell'" pyvista/
```

Although now that I'm taking another look, I only see 8 non-doc cell preference instances.
